spec/factories/mvi_profiles.rb

```diff
@@ -40,9 +40,9 @@
     given_names { Array.new(2) { Faker::Name.first_name } }
     family_name { Faker::Name.last_name }
     suffix { Faker::Name.suffix }
-    gender { Faker::Medical::Patient.gender }
+    gender { %w[M F].sample }
     birth_date { Faker::Date.between(from: 80.years.ago, to: 30.years.ago).strftime('%Y%m%d') }
-    ssn { Faker::Medical::SSN.ssn.delete('-') }
+    ssn { (1..1000).to_a.shuffle.join.to_s[0, 9] }
```

Comment:
```suggestion
    ssn { 9.times.map { rand(0..9) }.join }
```
☝️ Executes 50 times faster.

```
2.6.5 :018 > Benchmark.realtime { 10000.times { (1..1000).to_a.shuffle.join.to_s[0, 9] } }
 => 3.8803490000136662 
2.6.5 :019 > Benchmark.realtime { 10000.times { 9.times.map { rand(0..9) }.join } }
 => 0.08110899999155663
```